### DISCUSSION TITLE
Why can't I submit my application for Github Education Pack?

### DISCUSSION THREAD

**Discussion by 12944qwerty** [community/community#64963] [GitHub Education] [answered]
### Hi everyone,

I tried this with both chromium and firefox. I'm on the page where I have to submit proof of acceptance and I'm submitting my email letter of acceptance from my inbox. It does include the date in it.
When I do click Process, it finished the loading bar to 100% and then just refreshes the page. There is no other response after doing that. How can I submit it?

**Comment by advayc** [comment]
https://education.github.com/discount_requests/application

open this link to check if your application has gone through. If it has gone through then scroll down to check 'You've already submitted
1 request'. Your benefits will become active 72 hours after your application.

**Reply by 12944qwerty** [reply]
That redirects me to the same page where I have to upload the proof. https://education.github.com/discount_requests/11632182/additional_information

**Reply by HemanthSai7** [reply]
Exactly the same issue I'm facing. And every other link on GitHub redirects to this page. Previously I could view my applications and now whichever link I click takes me to this page where I have tried uploading so many times but it simply refreshes the page.

**Comment by arafajs** [comment]
Try submitting your application from a different device or browser.

**Reply by 12944qwerty** [reply]
I've tried firefox and chromium both. I will try a different device in the morning

**Reply by HemanthSai7** [reply]
I've tried it on windows (Edge) and now tried it on linux (Chrome). Still the same issue. My github is expiring in 5d and I'm unable to reverify.

**Comment by Noah-Haf** [comment]
Having the same issue too **(exactly the same behaviour)**. My github education is expiring in 11 days, whenever I upload my Image, I click submit, it loads to 100%, however, once it reaches 100% the page simply reloads and nothing happens. I've tried Edge, Chrome, Firefox and nothing works. When I visit https://education.github.com/discount_requests/application I am also redirected back to the page where I need to upload proof. 

Any ideas?

**Reply by arunsanjeevms** [reply]
Upload Proof in JPG Format. It works Well.

**Reply by dev-737** [reply]
> Upload Proof in JPG Format. It works Well.

It only lets me take a picture though...

**Comment by nqlp** [comment]
I'm also experiencing the same issue. I've tried using various browsers (Edge, Firefox) and even different operating systems (Windows and macOS), but the result remains unchanged. Whenever I click on 'Process my application,' the website consistently redirects me to https://education.github.com/discount_requests/11621382/additional_information. Consequently, I'm unsure whether my application has been received or not.

**Reply by arunsanjeevms** [reply]
Upload Proof in JPG Format. It works Well.

**Comment by ghost** [comment]
There is a bug here, even if you do not redirect to this page, when you re-upload the proof documents, you will not be able to pass because the photographed documents are not clear

**Comment by Noah-Haf** [comment]
I've tried again this evening; still no luck.

**Comment by ghost** [comment]
There is no other way, but to wait for the official bug fix, or add a function to upload pictures

**Comment by HemanthSai7** [comment, marked as answer]
Okay, So I uploaded my university ID card (by combining both sides of it into one). It got accepted. I guess it's only accepting univ ID cards.

**Reply by Noah-Haf** [reply]
I only have a letter with my proof of admission, it simply wont be feasible for me.

**Reply by nqlp** [reply]
I do have a card, but they are asking for an ID card that **includes a date**, and mine does not. So, I've sent them another document that does have a date included. Also, I don't think they are only accepting ID cards because they give two options: an ID card OR another form of proof.

**Reply by ghost** [reply]
Is your information a photo or not? I took a photo and uploaded a picture that is not clear

**Reply by HemanthSai7** [reply]
Yes, I know it should ideally accept both forms of proofs. I have tried with the admission letter before but it didn't work. ID worked just fine.

**Reply by 12944qwerty** [reply]
Great! Except for the fact that a physical ID costs $25 at my school :/

**Reply by nqlp** [reply]
Send them your statement of registration invoice / receipt. The most important is the **date** because it indicates your current school enrollment status.

**Reply by 12944qwerty** [reply]
I uploaded them my official letter of acceptance and same thing is happening

**Reply by nguyenthekhoig7** [reply]
This worked for me! I was trying with the front side of my Student ID and it kept on saying _Unable to submit your application. Please try again. I tried combining both sides of the ID and it was accepted immediately.

**Comment by nqlp** [comment]
I took a photo with my phone and it works!

**Reply by 12944qwerty** [reply]
Tried that, didn't work for me.

**Reply by Noah-Haf** [reply]
Gonna try this shortly.

**Comment by lucy2512** [comment]
I was using the github student developer pack for last 2 years, but still I am a student and I want to re-verify my student id and want to get the student developer pack benefits again. It is very much important for my projects. Previously I have requested for 9 times but every it got rejected now I am not able to upload or request further please help me. It is showing that you have outstanding error.
Please help me

**Comment by Noah-Haf** [comment]
Seems this issue is now resolved for me.

**Reply by nqlp** [reply]
How did you resolve it?

**Reply by Noah-Haf** [reply]
I used an OBS virtual camera, uploaded my evidence to the obs camera, took a photo and it worked, also make sure your github username is the  FULL NAME (no nicknames or anything) listed on your evidence.

**Comment by michael-722** [comment]
had this issue load no matter what I tried different browsers, forms of proof, different combinations of the proof literally everything. nothing worked would just reload the page and ask for the proof to be submitted again.

what fixed this was simply doing it on a phone instead of a laptop no clue why this fixed it but it did.

**Reply by Manmow** [reply]
Thank you! It worked on my phone. I completely don't know why it works on phone and not on pc. I used safari, so maybe that's why.

**Comment by josephko91** [comment]
I am facing the same issue. I tried uploading both sides of my ID with no success.

**Reply by Hanzlah10** [reply]
use phone to upload document. It worked for me !

**Comment by Hanzlah10** [comment]
It works on **phone**, I just did !

**Comment by zun43d** [comment]
It turns out the placement of front and back side of an ID card matters. I was facing the same issue. In my case, the validity date was on the back side of ID card. So I took two picture of my id card's front and back, then attached those in one photo. 

At first, in the image, I put the back side at left and front side at right. The application wasn't summitting. There were no error, its just after clicking on process, it refresh and brought back to the same page. I tried multiple times, but the same issue kept repeating.

Then I re-edited the image, but this time, I put front side at left and back side at right. This one was accepted and I was able to submit.

Hope this information helps someone.

**Comment by harshil3134** [comment]
I had the same issue. I tried about 10 times on different devices and browsers, but none of them worked. Finally, I realized that the problem was with the image of my document. If GitHub can't extract text from the image, it won't process it. So, try using a scanner and apply an effect to the image that will extract the text. This worked for me.

**Comment by Boden-C** [comment]
Had this issue and learned a few things:

1. If it redirects you to the same page, that means the photo failed, I'm guessing because it failed to extract any text
2. If you try and see the discount_requests/application and it redirects you, that means there is a current application with your account that did not submit. There is no limit to reupload, keep on trying until it works.

When it works, it will do a progress bar and redirect you to a success page. If you did not see that, it did not work, reupload again.

It is NOT
1. Only accepts ID cards (OFFICIAL ANSWER IS WRONG) - Got it working with an online Verification of Enrollment
2. Back and front for ID card - Got it working in the past with only the front, as long as there is the school name (maybe), name (and on billing), and date (absolutely important)
3. Browser (probably) - Got it working on multiple browsers

Maybe (needs confirmation) they made a mistake and only JPEG works not JPG? It might be a coincidence, but when I finally submitted my enhanced, it was .JPEG not .JPG. Their description says .JPEG, but it should be interchangeable.

**Comment by SaniaKiran** [comment]
My university is not listed in the GitHub student developer pack , how to list it >
